src/components/solution-control/solution-modify-dialog/solution-modify-dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+
+import { t } from '@i18n'
+import { Dialog } from '@ui/dialog'
+
+interface ISolutionModifyDialogProps {
```

Comment:
interface is the same as `ISolutionDeleteDialogProps`
You can just create a common component that would be receive all text stuff in props. No need to create two identical components for this.

Reply:
Done